app/views/includes/bulk_results_failed.scala.html

```diff
@@ -23,25 +23,30 @@
 @import views.html.helpers._
 
 <div class="divider--top clearfix">
-  <h3 id="failed-calculations">@Messages("gmp.bulk.subheaders.failedcalculations") (@bulkResultsSummary.failed)</h3>
+    <hr class="govuk-section-break govuk-section-break--visible">
+
+    <h3 class="govuk-heading-s" id="failed-calculations">@Messages("gmp.bulk.subheaders.failedcalculations") (@bulkResultsSummary.failed)</h3>
   <div class="column-three-quarters">
-    <ul class="bullets font-xsmall">
+    <ul class="govuk-list govuk-list--bullet font-xsmall">
       <li id="exp-failed-1" class="font-xsmall">@Messages("gmp.bulk.explanations.failed.1")</li>
       <li id="exp-failed-2" class="font-xsmall">@Messages("gmp.bulk.explanations.failed.2")</li>
       <li id="exp-failed-3" class="font-xsmall">@Messages("gmp.bulk.explanations.failed.3")</li>
     </ul>
   </div>
-  <div class="column-one-quarter text--right"><a id="download-failed" data-journey-click="gmp-bulk:click:download_failed" href="@routes.BulkResultsController.getResultsAsCsv(uploadReference, failed)">
-  @Messages("gmp.download.link")<span class="visuallyhidden"> @Messages("gmp.download.description.failed")</span></a></div>
+  <div class="column-one-quarter govuk-!-text-align-right"><a id="download-failed" data-journey-click="gmp-bulk:click:download_failed" href="@routes.BulkResultsController.getResultsAsCsv(uploadReference, failed)">
+  @Messages("gmp.download.link")<span class="hidden"> @Messages("gmp.download.description.failed")</span></a></div>
```

Comment:
should this class be "govuk-visually-hidden"?

Reply:
same as above